plotly/tools.py

```diff
@@ -1269,3 +1271,512 @@ def return_figure_from_figure_or_data(figure_or_data, validate_figure):
             )
 
     return figure
+
+### Quiver Plots ###
+def Quiver(X, Y, U, V, **kwargs):
+    """ Return data and layout objects for quiver plot
+    Example 1:
+    # X and Y are the coordinates of the arrow locations (created from numpy meshgrid),
+    # and U and V give the X and Y components of the arrow vectors. 
+    X,Y = np.meshgrid(np.arange(0,np.pi,.2),np.arange(0,np.pi,.2) )
+    U = np.cos(X)
+    V = np.sin(Y)
+    data, layout = Quiver(X,Y,U,V)
+    figure = Figure(data = data, layout = layout)
+    plot_url = py.plot(figure, filename = 'QuiverEx1')
+    Example 2:
+    X,Y = np.meshgrid(np.arange(0,2,.2),np.arange(0,2,.2))
+    U = np.cos(X)*Y;
+    V = np.sin(X)*Y;
+    data, layout = Quiver(X, Y, U, V)
+    figure = Figure(data = data, layout = layout)
+    plot_url = py.plot(figure, filename = 'QuiverEx2')
+
+    Keywords arguments with constant defaults:
+    angle (kwarg, angle in radians, default = np.pi/9):
+        Angle of arrowhead.
+    scale (kwarg, float in [0,1], default = .1):
+        Scales size of the arrows (avoid overlap).
+    arrow_length (kwarg, float in [0,1], default = 0.3):
+        Value multiplied to length of barb to get length of arrowhead.
+    color (kwarg, rgb vale, default = rgb(114, 132, 304)):
+        Set color of both barb and arrow.
+    barb_color (kwarg, rgb vale, default = color):
+        Set color of barbs.
+    arrow_color (kwarg, rgb vale, default = color):
```

Comment:
"value"
also, this can be any [valid html color string](http://www.w3.org/TR/css3-color/#colorunits), so we might want call the type "color string" instead of "rgb value"